.eslintrc

```diff
@@ -236,7 +236,8 @@
       "@web-stories-wp/eslint-import-resolver": {
         "mapping": {
           "^@googleforcreators\\/(.*)": "./packages/$1/src/",
-          "^@web-stories-wp\\/(.*)": "./packages/$1/src/"
+          "^@web-stories-wp\\/(.*)": "./packages/$1/src/",
+          "(.*)\\?url$": "./packages/moveable/src/$1"
```

Comment:
A bit surprised that you have to touch this config for the `?url` imports. Did you get "import not found" errors or something otherwise?

Ideally we don't need to touch this configuration at all...

Regarding `import/no-duplicates` we could probably just suppress that error if it's not a widespread issue

Reply:
I was also surprised that this was an issue. The error that shows without that lint mapping is `import/no-unresolved`:

<img width="846" alt="image" src="https://user-images.githubusercontent.com/22185279/154139450-ccc0f104-d129-4756-95ed-c4a7e2e27f7a.png">

Reply:
I see. Then it would be more appropriate to use [`eslint-import-resolver-webpack`](https://www.npmjs.com/package/eslint-import-resolver-webpack). 

Our custom eslint resolver just passes through to the default node resolver. It should not deal with stuff like `?url`

Let's remove this and suppress the ESLint error message instead.

Changing the resolver warrants its own ticket.